src/utils/insightsDescription.tsx

```diff
@@ -0,0 +1,92 @@
+import styled from "styled-components";
```

Comment:
Please move this file to `components/insights/InsightCard/InsightDescription/` and split it into `index.tsx`  and `styles.ts`

Also consider to create single `InsightDescription` component with the following signature:
```
const InsightDescription = (type: InsightType): JSX.Element | null 
```

It will have a icon with tooltip itself and `switch` statement inside of it and use it in `InsightCard` just like

```
<InsightDescription type={props.insightType}>
```